### PR TITLE
ci: Move integration-tests execution after deploy and add composite action for it

### PR DESCRIPTION
**Pull request by BenitoVisone**

### CODE REVIEW

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -21,8 +9,8 @@ on:
         required: true
         default: dev
         options:
-        - dev
-        - uat
+          - dev
+          - uat
```
Comment (giuseppe-gangemi) [inline .github/workflows/integration-tests.yml]: is this indentation correct? if so, [this file](https://github.com/pagopa/pdv-ms-tokenizer/blob/e23712244014f17989501d6388a7014f2baa532d/.github/workflows/integration-tests.yml#L12C14-L12C14) should be edited in the same way

Comment (BenitoVisone) [reply .github/workflows/integration-tests.yml]: I think both are correct, but for uniformity I'll set the same indentation

File: .github/workflows/integration-tests.yml [hunk 2]
```diff
@@ -21,8 +9,8 @@ on:
         required: true
         default: dev
         options:
-        - dev
-        - uat
+          - dev
```
Comment (BenitoVisone) [inline .github/workflows/integration-tests.yml]:
```suggestion
         - dev
```

File: .github/workflows/integration-tests.yml (hunk 1 above)
Comment (BenitoVisone) [inline .github/workflows/integration-tests.yml]:
```suggestion
         - uat
```

**Pull request merged**